### ISSUE TITLE
New machine, restored from backup: ssh keys don't work anymore.

### ISSUE THREAD

**Issue by ejheil** [blinksh/blink#762]
I have a few accounts set up to use SSH keys, on different hosts.  None of them seem to work anymore, and I'm not sure why.  I've tried deleting and recreating the host entry in configs, while keeping the same key.  I've tried creating a brand new key in blink shell.  I've copied the key using ssh-copy-id, and verified it showed up manually.  

This is what it looks like when I run ssh -v:

```
blink> ssh -v myhost
socket_callback_connected: Socket connection callback: 1 (0)
ssh_known_hosts_read_entries: Failed to open the known_hosts file '/etc/ssh/ssh_known_hosts': No such file or directory
blink: setting socket keepalive: 1
ssh_packet_userauth_failure: Access denied for 'none'. Authentication that can continue: publickey,password
blink: using auth methods attempt: 1 of 5
blink: warning: no key found: 'id_rsa' 
Password:
```

could the known hosts thing be an issue?

Thanks for any insight or additional debugging steps.

**Comment by elsbrock** [comment]
I am having the same issue. I suspect the key was actually not copied/restored, because when I click on „copy private key“ Blink just crashes.

**Comment by muammar** [comment]
I also am getting this type of error:

```
socket_callback_connected: Socket connection callback: 1 (0)
ssh_known_hosts_read_entries: Failed to open the known_hosts file '/etc/ssh/ssh_known_hosts': No such file or directory
blink: setting socket keepalive: 1
```

**Comment by carloscabanero** [comment]
Fixed on #364
